src/plugins/home/server/services/tutorials/tutorials_registry.ts

```diff
@@ -106,9 +126,16 @@ export class TutorialsRegistry {
     };
   }
 
-  public start() {
+  public start(core: CoreSetup | undefined, customIntegrations?: CustomIntegrationsPluginSetup) {
```

Comment:
discussed offline with @lukeelmers 

Will change this to use `CoreStart`. This will simplify the typing a lot.

Will continue to use the `CustomIntegrationsPluginsSetup` in the home-plugin start. The reason for this is that the beats-tutorials are added in `HomePlugin#start`, and we're not sure if this is actually 100% required. This is something we can address later down the road, and out of scope for this PR.